locale/circuitpython.pot

```diff
@@ -2158,7 +2166,7 @@ msgstr ""
 msgid "Size not supported"
 msgstr ""
 
-#: ports/atmel-samd/common-hal/alarm/SleepMemory.c
+#: ports/atmel-samd/common-hal/alarm.orig/SleepMemory.c
```

Comment:
I think changing the suffix of the file will make the translation scripts ignore it. (And therefore leave this as-is.)

Reply:
> @tannewt Sure I'll kick the tires on this PR tonight.
> 
> @t-ikegami your comment adding edge support for PinAlarm reminded me of this SAMD5x errata. Did you run into this? <img alt="" width="400" src="https://user-images.githubusercontent.com/29153441/154559182-16533ef8-e47f-46fd-a426-d34bdd3edab6.png">

I have tried to check the errata condition, by using a Wio Terminal in hand and PA02 pin.  As far as wake up from BACKUP sleep is concerned, the tamper pin interrupt is triggered only for the transition from high/low to Hi-Z state, irrelevant to TAMLVL setting.  The behavior disagrees with the datasheet/errata, though it seems to be reasonable as a tamper detection.  Note that, while not in sleep mode (or IDLE sleep, maybe), the TAMPID register reacts as expected: if we set TAMLVLx = 0, TAMPID is flagged only on the falling edge.  Honestly, I'm confused, and hope someone to make a double check.